### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by cstenglein**
Various improvements and some cleanup

### CODE REVIEW

File: src/components/Apps/AppInfo/AppInfo.tsx [hunk 1]
```diff
@@ -31,25 +30,31 @@ export const AppInfo: FC<Props> = ({ app, onClose }) => {
...
+          .catch((_) => {
+            // do nothing if cloud image doesn't exist either
+          });
       });
 
     // import app images
-    [1, 2, 3].forEach((num) => {
-      import(`../../../assets/apps/preview/${id}/${num}.png`)
-        .then((image) => {
+    Promise.allSettled([
```
Comment (escapedcat) [inline src/components/Apps/AppInfo/AppInfo.tsx:41]: Nitpick: Is it possible to asign this to a var and use await? Like i.e. here: https://dmitripavlutin.com/promise-all-settled/#21-all-promises-fulfilled  
Might be easier to read.

File: src/container/ConnectStatus/ConnectStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { FC, useContext } from "react";
```
Comment (escapedcat) [inline src/container/ConnectStatus/ConnectStatus.tsx:1]: Nitpick: Wanna seperate this?  
```ts
import { useContext } from "react";
import type { FC } from "react";
```

**Review by escapedcat — APPROVED** [review]

File: src/components/Apps/AppInfo/AppInfo.tsx (hunk 1 above)
Comment (cstenglein) [reply src/components/Apps/AppInfo/AppInfo.tsx]: Okay, done :)

File: src/container/ConnectStatus/ConnectStatus.tsx (hunk 2 above)
Comment (cstenglein) [reply src/container/ConnectStatus/ConnectStatus.tsx]: Done

**Review by escapedcat — APPROVED** [review]

**Pull request merged**